src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -155,7 +155,7 @@ internal static RuntimeGraph GetRuntimeGraph(IEnumerable targetFrameworks)
         internal static NuGetFramework GetTargetFramework(IVsProjectProperties properties, string projectFullPath)
         {
             var targetFrameworkMoniker = GetPropertyValueOrNull(properties, ProjectBuildProperties.TargetFrameworkMoniker);
-            var targetPlatformMoniker = GetPropertyValueOrNull(properties, ProjectBuildProperties.TargetPlatformIdentifier);
+            var targetPlatformMoniker = GetPropertyValueOrNull(properties, ProjectBuildProperties.TargetPlatformMoniker);
```

Comment:
Can you pelase add a test. 

We didn't have any tests for these scenarios and that's why the bug was introduced. 

Let's cover that gap: https://github.com/NuGet/NuGet.Client/blob/e97d556a83750bb4f8017e3640bc699d04832657/test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs#L73

You can use that as a template...just provide a net50-win123 framework.